### PR TITLE
Added nickname to random name generator

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
#### Summary

```SUMMARY: Features "Added nickname to random name generator"```

#### Purpose of change

Allow one in three chance for random full name to have a nickname selected from the list.

#### Additional context

Also added 220 English nicknames for the start.

image

### CODE REVIEW

File: data/names/en.json [hunk 1]
```diff
@@ -1,4 +1,233 @@
...
+{"usage": "nick", "name": "Apple"},
+{"usage": "nick", "name": "Appler"},
+{"usage": "nick", "name": "Arkansawyer"},
+{"usage": "nick", "name": "Arkie"},
+{"usage": "nick", "name": "Arky"},
+{"usage": "nick", "name": "Aussie"},
+{"usage": "nick", "name": "Australian"},
+{"usage": "nick", "name": "Bald"},
+{"usage": "nick", "name": "Banana"},
+{"usage": "nick", "name": "Banana"},
```
Comment (Night-Pryanik) [inline data/names/en.json]: Duplicate.

File: data/names/en.json [hunk 2]
```diff
@@ -1,4 +1,233 @@
...
+{"usage": "nick", "name": "Cabbage"},
+{"usage": "nick", "name": "Cajun"},
+{"usage": "nick", "name": "Canada"},
+{"usage": "nick", "name": "Canadian"},
+{"usage": "nick", "name": "Candango"},
+{"usage": "nick", "name": "Canuck"},
+{"usage": "nick", "name": "Capixaba"},
+{"usage": "nick", "name": "Carcamano"},
+{"usage": "nick", "name": "Carioca"},
+{"usage": "nick", "name": "Categories:"},
```
Comment (Night-Pryanik) [inline data/names/en.json]: Hmm?

File: data/names/en.json [hunk 3]
```diff
@@ -1,4 +1,233 @@
...
+{"usage": "nick", "name": "Export"},
+{"usage": "nick", "name": "Flatlander"},
+{"usage": "nick", "name": "Foolio"},
+{"usage": "nick", "name": "France"},
+{"usage": "nick", "name": "French"},
+{"usage": "nick", "name": "From"},
+{"usage": "nick", "name": "Fudgie"},
+{"usage": "nick", "name": "Gamma"},
+{"usage": "nick", "name": "Gaucho"},
+{"usage": "nick", "name": "Gaúcho"},
```
Comment (Night-Pryanik) [inline data/names/en.json]: Almost identical with the `Gaucho`?

File: data/names/en.json [hunk 4]
```diff
@@ -1,4 +1,233 @@
...
+{"usage": "nick", "name": "Geordie"},
+{"usage": "nick", "name": "German"},
+{"usage": "nick", "name": "Germany"},
+{"usage": "nick", "name": "Gold"},
+{"usage": "nick", "name": "Goober"},
+{"usage": "nick", "name": "Gray"},
+{"usage": "nick", "name": "Greece"},
+{"usage": "nick", "name": "Greek"},
+{"usage": "nick", "name": "Green"},
+{"usage": "nick", "name": "Green"},
```
Comment (Night-Pryanik) [inline data/names/en.json]: Duplicate.

File: data/names/en.json [hunk 5]
```diff
@@ -1,4 +1,233 @@
...
+{"usage": "nick", "name": "Iron"},
+{"usage": "nick", "name": "Irving"},
+{"usage": "nick", "name": "Italian"},
+{"usage": "nick", "name": "Italy"},
+{"usage": "nick", "name": "Jackeen"},
+{"usage": "nick", "name": "Jafa"},
+{"usage": "nick", "name": "Janner"},
+{"usage": "nick", "name": "Jarocho"},
+{"usage": "nick", "name": "Juice"},
+{"usage": "nick", "name": "Juice"},
```
Comment (Night-Pryanik) [inline data/names/en.json]: Duplicate.

File: data/names/en.json [hunk 6]
```diff
@@ -1,4 +1,233 @@
...
+{"usage": "nick", "name": "Jump"},
+{"usage": "nick", "name": "Kappa"},
+{"usage": "nick", "name": "Killer"},
+{"usage": "nick", "name": "Kiwi"},
+{"usage": "nick", "name": "Kraut"},
+{"usage": "nick", "name": "Lambda"},
+{"usage": "nick", "name": "Languages"},
+{"usage": "nick", "name": "Leodiensian"},
+{"usage": "nick", "name": "List"},
+{"usage": "nick", "name": "Loiner."},
```
Comment (Night-Pryanik) [inline data/names/en.json]:
```suggestion
{"usage": "nick", "name": "Loiner"},
```

Maybe Loner?

File: data/names/en.json [hunk 7]
```diff
@@ -1,4 +1,233 @@
...
+{"usage": "nick", "name": "Juice"},
+{"usage": "nick", "name": "Juice"},
+{"usage": "nick", "name": "Juicy"},
+{"usage": "nick", "name": "Jump"},
+{"usage": "nick", "name": "Kappa"},
+{"usage": "nick", "name": "Killer"},
+{"usage": "nick", "name": "Kiwi"},
+{"usage": "nick", "name": "Kraut"},
+{"usage": "nick", "name": "Lambda"},
+{"usage": "nick", "name": "Languages"},
```
Comment (Night-Pryanik) [inline data/names/en.json]: Hmm?

File: data/names/en.json [hunk 8]
```diff
@@ -1,4 +1,233 @@
...
+{"usage": "nick", "name": "Platinum"},
+{"usage": "nick", "name": "Polentone"},
+{"usage": "nick", "name": "Pom"},
+{"usage": "nick", "name": "Porteño"},
+{"usage": "nick", "name": "Print"},
+{"usage": "nick", "name": "Psi"},
+{"usage": "nick", "name": "Raggie"},
+{"usage": "nick", "name": "Red"},
+{"usage": "nick", "name": "Redneck"},
+{"usage": "nick", "name": "References"},
```
Comment (Night-Pryanik) [inline data/names/en.json]: Hmm?

**Comment by thquinn** [comment]
Cool! One in three seems pretty high to me, but I wonder what others think. At the very least, it's a good way to get people starting to see them.

I'll definitely be making a PR with more when this goes in. :)

**Comment by kevingranade** [comment]
Tests failing and stuff.
Because we have name tests? neat.

**Comment by ZhilkinSerg** [comment]
Tests were updated.

**Pull request merged**